tests/test_text.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+"""\
+This is a python port of "Goose" orignialy licensed to Gravity.com
+under one or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.
+
+Python port was written by Xavier Grangier for Recrutae
+
+Gravity.com licenses this file
+to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import unittest
+
+from goose3.text import get_encodings_from_content
+
+
+class TestText(unittest.TestCase):
+
+    def test_get_encodings_from_content(self):
+        self.assertEqual(
+            get_encodings_from_content(
+                '<meta charset=utf-8 " />'),
+            ['utf-8']
+        )
+        self.assertEqual(
+            get_encodings_from_content(
+                '<meta http-equiv="content-type" content="text/html; charset=utf-8 " />'),
+            ['utf-8']
+        )
+        self.assertEqual(
+            get_encodings_from_content(
+                '<meta http-equiv="content-type" content="text/html; charset=utf-8    " />'),
+            ['utf-8']
+        )
+        self.assertEqual(
+            get_encodings_from_content(
+                b'<meta http-equiv="content-type" content="text/html; charset=utf-8 " />'),
+            [b'utf-8']
```

Comment:
```python
In [1]: s = b'abc'

In [2]: s.decode('utf-8')
Out[2]: 'abc'

In [3]: s.decode(b'utf-8')
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-3-eedbddf143a8> in <module>()
----> 1 s.decode(b'utf-8')

TypeError: decode() argument 1 must be str, not bytes
```

encoding should be `str` on Python 3

https://github.com/requests/toolbelt/blob/f6d3afbfc53dc8f83efcb92f2b72e44c99d44c8e/requests_toolbelt/utils/deprecated.py#L39-L40